open-hackathon-server/src/hackathon/hk8s/k8s_service_adapter.py

```diff
@@ -44,12 +44,20 @@ def __init__(self):
         return
 
     def create_k8s_deployment_with_yaml(self, yaml, name, namespace):
-        k8s_api = utils.create_from_yaml(self.k8s_client, yaml)
-        resp = k8s_api.read_namespaced_deployment(name, namespace)
-        return str(resp.status)
+        try:
+            k8s_api = utils.create_from_yaml(self.k8s_client, yaml)
```

Comment:
do we need to cache the `k8s_api`? Looks like we don't have to create it from yaml every time

Reply:
Can simply add `todo` for now.

Reply:
need to investigate further, todo for now

Reply:
Add a TODO accordingly, will look into it later.